src/contribution/projectsHome.js

```diff
@@ -0,0 +1,58 @@
+window.c.contribution.projectsHome = (function(m, c){
+  return {
+
+    controller: function() {
+      var vm = {recommendedCollection: m.prop([]),
+                recentCollection: m.prop([]),
+                expiringCollection: m.prop([])
+          },
+          expiring = m.postgrest.filtersVM({origin: 'eq'});
+          expiring.origin('expiring');
+          recommended = m.postgrest.filtersVM({origin: 'eq'});
+          recommended.origin('recommended');
+          recents = m.postgrest.filtersVM({origin: 'eq'});
+          recents.origin('recents');
+
+      c.models.projectsForHome.getPage(1, recommended.parameters()).then(function(data){
```

Comment:
Let's work to use only the projects endpoint. We should get rid of projects_for_home. Projects can repeat in the home page.